### PR TITLE
Add plugin support for group and improve readability in installing

### PR DESCRIPTION
**Pull request by kreddyj**
**What this PR does / why we need it**:
* Add plugin support for group
* improve readability in output while installing

### CODE REVIEW

File: build/purser-install.sh [hunk 1]
```diff
@@ -24,16 +24,22 @@ cd $HOME/opt/purser-env
 # === Purser Controller ===
 
 # Get Cluster config file location
-read -p "Taking $HOME/.kube/config as your cluster's configuration (yes/no): " isDefaultConfig
-if [ "$isDefaultConfig" != "yes" ]
+read -p "Enter your cluster's configuration. (Leave blank for default $HOME/.kube/config): " readConfig
```
Comment (gurusreekanthc) [inline build/purser-install.sh]: configuration path?
>>Leave blank for default.
This is not needed. Default behavior is pressing enter command if it is same in any installation.

**Comment by hemani19** [comment]
Please also paste the screenshots of the new output in the PR description.

**Comment by kreddyj** [comment]
@gurusreekanthc @hemani19 Made changes based on your comments. Also tested plug-in support for getting group details.

**Comment by kreddyj** [comment]
Merging the branch to make a demo release. We can revert this commit if there is a bug.

**Pull request merged**